src/daemon/daemon.h

```diff
@@ -60,7 +64,7 @@ struct MetricsOptInData
 };
 
 struct DaemonConfig;
-class Daemon : public QObject, public multipass::Rpc::Service, public multipass::VMStatusMonitor
+class Daemon : public QObject, public multipass::VMStatusMonitor
```

Comment:
youch, good catch

Reply:
Yeah, it was cruft left over from when we split out the `DaemonRpc` class and should have been removed then.  But better late than never:)